app/javascript/partials/_nested_forms.js

```diff
@@ -83,6 +88,38 @@ $(function() {
     ...defaultNestedFormsOptions,
     beforeAddForm: ($container) => {
       $container.children('.nested-medium-form').removeClass('d-none');
+    },
+    afterAddForm: function(_, $form) {
+
+      // If we only have one nested media form, we don't have to do anything
+      // becasue the index of 0 is already the right one
+      if ($('.nested-medium-form').length === 1) {
+        return;
+      }
```

Comment:
this is causing the bug opened with SVA-626, because after deleting all existing file sections and starting over again with adding one, functionality is not bind here. it was only working again after adding at least a second file section.

Reply:
done with #181